### DISCUSSION TITLE
jsonrpc: 2.0 error

### DISCUSSION THREAD

**Discussion by edos01** [Cyfrin/foundry-full-course-cu#705] [General]
Hi guys i keep getting this error when Deploying my contract in solidity Injected Provider - Metamask

creation of FundMe pending...
creation of FundMe errored: Returned error: {"jsonrpc":"2.0","error":"The requested account and/or method has not been authorized by the user.","id":1649751740533797}

BUT it shows that my transaction has successfully gone through on Etherscan.... I am using SepoliaETH test network.

**Comment by theirrationalone** [comment]
Please share a ss of remix and its terminal...

**Reply by edos01** [reply]
thanks for the response mate,

![2](https://github.com/Cyfrin/foundry-full-course-f23/assets/16897339/c582164c-cdcc-4201-9470-5dc27ac927b7)
![Screenshot 2023-08-25 083618](https://github.com/Cyfrin/foundry-full-course-f23/assets/16897339/48498037-0612-4b8f-9b33-32edfa440885)

**Reply by edos01** [reply]
And this ss is for etherscan showing that the transaction has been made


![3](https://github.com/Cyfrin/foundry-full-course-f23/assets/16897339/bd86f79d-b10c-4643-8299-51e7abbfd577)

**Comment by theirrationalone** [comment]
It might be a glitch, timeout timer reached due to heavy data fetching... And wrong states executed & taken into account i.e., last stable permission state, last account connection... Etc. 

Please try to clear local storage of your remix website. 

You can follow this discussion to do that... [706](https://github.com/Cyfrin/foundry-full-course-f23/discussions/706)

**Reply by edos01** [reply]
Thanks mate let me try that
